klarna-on-demand/src/main/java/com/klarna/ondemand/UrlHelper.java

```diff
@@ -14,8 +14,12 @@ static String baseUrl() {
         return KlarnaProductionUrl;
     }
 
-    static String registrationUrl() {
-        return String.format("%s/registration/new?api_key=%s&locale=%s", baseUrl(), Context.getApiKey(), defaultLocale());
+    static String registrationUrl(android.content.Context context) {
+        return String.format("%s/registration/new?api_key=%s&locale=%s&public_key=%s",
+                baseUrl(),
+                Context.getApiKey(),
```

Comment:
There are two "context"s here, ours and the app's. This is highly confusing and we may want to use our context's fully qualified name to mitigate this.